docs/tutorials/getting_started/command_line.rst

```diff
@@ -38,9 +37,9 @@ Here are some of the plots that are output when running with the latest total nu
 
 The results from the tournament itself (ordered by median score):
 
-.. image:: http://axelrod-python.github.io/tournament/assets/strategies_boxplot.svg
-   :width: 50%
-   :align: center
```

Comment:
Could you help me understand _why_ the tournament repo is separate please? I think that's what this comes down to